build.cmd

```diff
@@ -178,6 +178,9 @@ REM === Start the build steps
 REM ===
 REM =========================================================================================
 
+echo %__MsgPrefix%Using environment: "%__VSToolsRoot%\VsDevCmd.bat"
```

Comment:
Is this required for run.cmd? If so, should it go within run.cmd @Priya91 ?

That said, we should use "setlocal" to localize the environment setup by this?